src/main/java/ch/uzh/ifi/hase/soprafs22/service/LobbyService.java

```diff
@@ -141,4 +149,34 @@ public ILobby getLobby(String token, Long lobbyId){
         throw new ResponseStatusException(HttpStatus.FORBIDDEN, errorMessage);
     }
 
+    public void updateLobby(ILobby lobby, String token, String lobbyName, Visibility visibility, GameMode gameMode, GameType gameType) {
+        if (!token.equals(lobby.getHost().getToken())) {
+            throw new ResponseStatusException(HttpStatus.UNAUTHORIZED, "User is not the host of the lobby.");
+        }
+
+        // Check if values are valid
+        checkLobbyNameNullOrEmpty(lobbyName, "updated");
+        checkEnumConfigNull(visibility, "visibility", "updated");
+        checkEnumConfigNull(gameMode, "game mode", "updated");
+        checkEnumConfigNull(gameType, "game type", "updated");
```

Comment:
Does this mean that the client now has to provide everything (visibility, game mode, etc.) to update the lobby name?